app/scripts/components/common/cookie-consent/index.tsx

```diff
@@ -0,0 +1,127 @@
+import React, { useState, useEffect } from 'react';
+
+import {
+  Alert,
+  Button,
+  ButtonGroup,
+  Link,
+  Icon
+} from '@trussworks/react-uswds';
```

Comment:
we have a `uswds` directory where instead of importing components directly from `uswds`, we create an abstraction layer and import them from this directory instead. This would help us visually identify right away what components we depend on from the DS. To decouple the DS component(s) quicker if need be. If any breaking changes is updated in the DS, we have the ability to just handle it in one place rather than multiple where direct imports are happening. Looks like we haven't been keeping to that standard though. Still early on, so i recommend we do.